src/dectris/neggia/data/H5HeaderMsg.h

```diff
@@ -26,25 +26,20 @@ SOFTWARE.
 #define H5HEADERMSG_H
 #include "H5Object.h"
 
-/// https://www.hdfgroup.org/HDF5/doc/H5.format.html#ObjectHeaderMessages
+/// https://support.hdfgroup.org/HDF5/doc/H5.format.html#ObjectHeaderMessages
 /// Contains type, size and flags of Header Message part of H5ObjectHeader:
-/// https://www.hdfgroup.org/HDF5/doc/H5.format.html#ObjectHeaderPrefix
+/// https://support.hdfgroup.org/HDF5/doc/H5.format.html#V1ObjectHeaderPrefix
 
-class H5HeaderMsgPreamble: public H5Object
-{
+class H5HeaderMsgWrapper {
 public:
-   H5HeaderMsgPreamble() = default;
-   H5HeaderMsgPreamble(const char * fileAddress, size_t offset);
-   H5HeaderMsgPreamble(const H5Object & other);
-   H5Object getHeaderMsg() const;
-   uint16_t type() const;
-   uint16_t size() const;
-   uint8_t flags() const;
-   static constexpr int DATA_OFFSET = 8;
+    H5HeaderMsgWrapper(const H5Object& headerMessage, uint16_t messageType);
+    H5Object getHeaderMsg() const;
+    uint16_t type() const;
 
 private:
-   void _init();
-
+    H5HeaderMsgWrapper() = delete;
+    H5Object _headerMessage;
+    uint16_t _messageType;
 };
```

Comment:
We could consider making this a simple struct.  It basically does nothing but hold fields. What about using the name `H5HeaderMsgType` since now its basically just a tuple of a `H5HeaderMsg` and type?

Reply:
sounds good!

Reply:
the "move methods to private" change got lost due to the reformat stuff. 
it's back in!

Reply:
Or maybe the name `H5HeaderMsg` is even better? What about changing the field names to `message` and `type`?